api/src/main/java/com/valtech/aem/saas/api/caconfig/SearchConfiguration.java

```diff
@@ -53,4 +54,8 @@
     @Property(label = "Autocomplete options Max Limit",
               description = "The maximum number of autocomplete options displayed.")
     int autocompleteOptionsMax() default DEFAULT_AUTOCOMPLETE_RESULTS_MAX_LIMIT;
+
+    @Property(label = "Enable Tracking",
+              description = "Flag that enables click tracking feature in the search component.")
+    boolean enableTracking() default DEFAULT_ENABLE_TRACKING;
```

Comment:
- [ ] please change to "enableSearchResultItemTracking" in all methods, contstants and comments to be very clear about the feature